packages/core/src/database/utils.ts

```diff
@@ -52,7 +52,7 @@ export const convertToPrimitiveOrSql = (
     return value;
   }
 
-  throw new Error(`Cannot convert ${key} with ${value.toString()} to primitive`);
```

Comment:
what's the consideration of removing value here?

Reply:
besides array， the remaining value type is never... which can not convert to string. I believe we have all types covered above.